scripts/skills/actives/rf_en_garde_toggle_skill.nut

```diff
@@ -144,9 +144,10 @@ this.rf_en_garde_toggle_skill <- ::inherit("scripts/skills/skill", {
 		}
 	}
 
-	function onAffordablePreview( _skill, _movementTile )
+	function onAfterUpdate( _properties )
 	{
-		this.modifyPreviewField(this, "FatigueCost", 15, false);
+		if (this.getContainer().getActor().isPreviewing())
+			this.m.FatigueCost = 15;
```

Comment:
Because En Garde requires at least 15 fatigue available to trigger, this is a hacky way to inform the player about whether their previewed action will put them over that threshold.

Reply:
Then you should
- use the variable `FatigueRequired` that you already declared in this file
- add a comment here explaining this exact thing about the "hacky way to inform the player"

Reply:
Sounds good, will add that.